src/connectors/impls/elastic.rs

```diff
@@ -54,6 +82,10 @@ fn default_concurrency() -> usize {
     DEFAULT_CONCURRENCY
 }
 
+fn default_auth() -> Auth {
```

Comment:
could we make Auth 'Default'? that way we avoid extra functions for this

Reply:
Done